tools/setup_riak.escript

```diff
@@ -1,6 +1,12 @@
 #!/usr/bin/env escript
 -mode(compile).
 
+-ifdef(OTP_RELEASE).
```

Comment:
Create `backward_compatible.hrl` and put all functions for backward compatible 22 <-> 23 <-> 24 OTP's, eg `erlang:get_stacktrace/1`, `crypto:hmac/3`, `ssl:ssl_accept/2`, `pg2:get_members/1`, `pg2:create/1`, `pg2:join/2` etc...

Reply:
For the case of `get_stacktrace`, I don't think we have any OTP version that doesn't support the newer way to do it, so we can just do it the new way, right? 🤔

Reply:
The `riak` image is based on old version(as I understand), by this reason I was add this logic here(the new style was used first), please see CT error in https://github.com/esl/MongooseIM/runs/3071190966?check_suite_focus=true for `riak` OTP 23.

Reply:
Huh, so I'd rather update that container 😕